lib/tasks/active_record_views.rake

```diff
@@ -52,12 +52,46 @@ Rake::Task[schema_rake_task].enhance do
     pg_tasks = tasks.send(:class_for_adapter, adapter).new(config)
     pg_tasks.send(:set_psql_env)
 
-    require 'shellwords'
-    system("pg_dump --data-only --table=active_record_views #{Shellwords.escape database} >> #{Shellwords.escape filename}")
-    raise 'active_record_views metadata dump failed' unless $?.success?
+    begin
+      active_record_views_dump = Tempfile.open("active_record_views_dump.sql")
+      require 'shellwords'
+      system("pg_dump --data-only --no-owner --table=active_record_views #{Shellwords.escape database} >> #{Shellwords.escape active_record_views_dump.path}")
```

Comment:
I was wondering if it’s worth adding any coverage for the owner or comment removal. I can’t think of any good positive assertions to add to the “copies over activerecord_views data” test. Probably not worth testing the owner bit since that’s psql behaviour and if call into to Rails code, I think not testing `remove_sql_header_comments` is fine too.

Reply:
I'm leaning towards “no” for tests. It's quite low risk because the biggest thing we're likely to break is someones schema dump file, which should be obvious in the users dev/test environment.